uktrade/data-hub-frontend
# Prevent acitvity tab from crashing for interactions without a service name

## Description of change

There are interactions with no service which error and prevent interactions from loading on the activity tab for companies. These interactions do not crash on the main interactions page which shows all interactions.

## Test instructions

See current issue on [staging](https://staging.datahub.uktrade.digital/companies/b2c34b41-1d5a-4b4b-9249-7c53ff2868dd/activity)

You can search for the same interaction on the all interactions page and see it loads fine on that collectionlist.

Checkout this branch and point your frontend to staging and load the mars export ltd company.
http://localhost:3000/companies/b2c34b41-1d5a-4b4b-9249-7c53ff2868dd/overview

Go to the activity page and see the activity tab loads as expected.

## Screenshots

### Before

<img width="1051" alt="Screenshot 2024-08-21 at 16 07 06" src="https://github.com/user-attachments/assets/92296fa5-0b7d-4b28-9f35-211a30d0c91e">

### After

<img width="1051" alt="Screenshot 2024-08-21 at 16 09 15" src="https://github.com/user-attachments/assets/ffcdbe9b-1721-4136-8ccc-d0c79e613f20">

## Checklist

[//]: # 'When submitting a PR make sure the code review guidelines have been satisfied.
https://github.com/uktrade/data-hub-frontend/blob/main/docs/Code%20review%20guidelines.md'
